### PR TITLE
geodata from dc area and michigan

### LINKED ISSUE #212: geodata from DC metro area for map

### PR DESCRIPTION
**Pull request by HP4k1h5**
Description
-----------
data from the dc metro area including 
- baltimore
- northern va
- dc

Motivation
----------
expand db of sites

Testing Guidelines
------------------

Release Checklist
-----------------

- [ ] This code has unit tests
- [ ] I have a plan to verify that these changes are working as expected after deploy
- [ x] I have a plan for how to revert, rollback, or disable these changes if things are not working as expected

Security Checklist
-------------------

This PR creates, modifies, or deletes:

- [ ] API routes: API routes, parameters, or user authorization
- [ ] Authentication: Authentication mechanism
- [ ] Credentials: Server side credentials, or secrets in configuration / source code
- [ ] Cryptography: Encryption, hashing, certificates, signatures, random numbers, etc.
- [ x] User data: personal information handling, logs, error messages, etc.

Additional Notes
----------------
closes #212

### CODE REVIEW

File: map/src/data/markers.ts [hunk 1]
```diff
@@ -263,17 +271,23 @@ const LOCATIONS = {
...
+      lng: -79.122097,
+      serviceRadius: 32670,
+    },
     VA_NORFOLK: {
       description: 'Norfolk, Virginia',
       lat: 36.846224,
       lng: -76.284361,
       serviceRadius: 32670,
     },
-    VA_AUGUSTA: {
```
Comment (sabind) [inline map/src/data/markers.ts]: We  shouldn't be removing existing locations as other map points do depend on them.

File: map/src/data/markers.ts [hunk 2]
```diff
@@ -1070,6 +1084,659 @@ This Mutual Aid Network is a grassroots, community focused and lead ecosystem fo
...
+    getHelp: { email: [], web: {} },
+    volunteers: {
+      email: [],
+      web: {
+        Volunteer:
+          'https://docs.google.com/document/d/1FYcqcbY2qMuuy75GIIM9UuqvFR_-YlVKfFcrqJOy9mw/edit?fbclid=IwAR1Lfd0mZrm4GgqOwmcXAeJGnTLUEQ0wmooezGvgnVYjXvKEZT8EYPU2tUI',
+      },
+    },
+    contentBody: '',
+    loc: {
```
Comment (sabind) [inline map/src/data/markers.ts]: We should pull these locations out as specific locations. It looks like it's smaller towns within Michigan.

File: map/src/data/markers.ts [hunk 3]
```diff
@@ -263,17 +271,23 @@ const LOCATIONS = {
       lng: -72.717293,
       serviceRadius: 132230,
     },
+    VA_AUGUSTA: {
```
Comment (HP4k1h5) [inline map/src/data/markers.ts]: moved here

File: map/src/data/markers.ts (hunk 1 above)
Comment (HP4k1h5) [reply map/src/data/markers.ts]: sorry it got moved here https://github.com/reach4help/reach4help/pull/213/files/4f6ea499bdd61b202eb6d2e649b31572038ea1bd#r400551469, i was trying to alphabetize this so folks dont duplicate

File: map/src/data/markers.ts (hunk 2 above)
Comment (HP4k1h5) [reply map/src/data/markers.ts]: ok, i scraped these from another data source, ill see if i can get some better location descriptions

Comment (sabind) [reply map/src/data/markers.ts]: Ok sounds good. ^

Comment (HP4k1h5) [reply map/src/data/markers.ts]: i did find an api to convert the latlng to reasonable place names, thanks for looking this over.
please let me know what else i can do. cheers

Comment (sabind) [reply map/src/data/markers.ts]: this is pretty excellent honestly.

**Comment by HP4k1h5** [comment]
> kk just remove the `yarn.lock` file and check out the conflicts and this should be good to go.

can i add `yarn.lock` to `.gitignore` ?

File: map/src/data/markers.ts (hunk 2 above)
Comment (HP4k1h5) [reply map/src/data/markers.ts]: thank you. i think i resolved the conflicts. much appreciated for your patience

**Comment by s0** [comment]
Hi @HP4k1h5, @sabind meant that the PR shouldn't modify `yarn.lock`, not that it should delete it from the tree :) If you can update your PR to not make any changes to this file then we'l be good to go! :)

**Comment by HP4k1h5** [comment]
> Hi @HP4k1h5, @sabind meant that the PR shouldn't modify `yarn.lock`, not that it should delete it from the tree :) If you can update your PR to not make any changes to this file then we'l be good to go! :)

got it, thank you. my apologies for the back and forth. your help is very much appreciated. 
its my first time interacting with a lot of the dev tools, but kudos to the team for standing up a great project so quickly.

**Comment by sabind** [comment]
@HP4k1h5 - Resolved a conflict and merged!

Thank you again for your contributions! Keep it up 👍

**Pull request merged**